### ISSUE TITLE
Workspaces card for Account Dashboard

### ISSUE THREAD

**Issue by catrobson** [openshiftio/openshift.io#2848]
As part of the new Account Home Dashboard work, new cards were designed. The dashboard has already been created in HTML/CSS (issue #925 and PR fabric8-ui/fabric8-ui#2662), but is not yet hooked up. This will need to be done before it can be pushed to production and flags removed.

Verification:
- [ ] Workspaces card matches designs
- [ ] Internal Feature Flag and Development Flags removed from all components used by the Account Home Dashboard

**Comment by mindreeper2420** [comment]
@ldimaggi @ljelinkova @ppitonak This is going to be one of the new cards on the Home Dashboard (_home).

**Comment by catrobson** [comment]
Closing as duplicate of https://github.com/openshiftio/openshift.io/issues/2877
